### PR TITLE
Adding dockerfile and .dockerignore - this is now building!

### PR DESCRIPTION
**Pull request by sponte**

### CODE REVIEW

**Comment by sponte** [comment]
if you'd like to try locally, install docker toolbox and in the root:

```
docker build -t specs .
docker run -Pd --name specs specs
```

**Comment by jimCresswell** [comment]
I will try it first thing tomorrow :)

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ADD . /app
+
+WORKDIR /app
+
+RUN npm install
+
+EXPOSE 3000
+VOLUME /app/project-data
+
+ENV SPECS_ALLOW_INSECURE_SSL false
```
Comment (jimCresswell) [inline Dockerfile]: This is very cool. Do we need another environment variable to set the storage directory or is the local /project-data already mapped to the external volume?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM node:0.12.4
```
Comment (jimCresswell) [inline Dockerfile]: I ran the app against Node 4 yesterday, no problems.

File: .dockerignore [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# Project files (for now)
```
Comment (jimCresswell) [inline .dockerignore]: If this is going to map to .gitignore is there a way to do that automatically?

**Comment by jimCresswell** [comment]
Well, this is amazing. I got it to run, I'm going to merge.

**Comment by jimCresswell** [comment]
@sponte actually, it needs a version bump :smile:

**Comment by jimCresswell** [comment]
I'll do it...

**Pull request merged**